### DISCUSSION TITLE
fix if you cant access the json

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#369] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
ok so this will work on chromebooks only one v108 which i assume most of you are on
here are the steps

step 1) cope

step 2) realize that risking all this stuff, just to play a game on your school computer, is pretty useless. getting in trouble for getting caught doing this unblocking shits that doesnt work 90% of the time isnt worthless. explaining to your parents why you got in trouble isnt worth it, and realizing your education is important too.

step 3) [https://docs.google.com/document/d/1PyjkcA4CVbXYX4gYIh1gM6wPz1SDFr179ReIonT5Q5k/edit#](url)

**Comment by obkduishinijf** [comment]
don't work for some people, even on v108 because full access to add another account.. is dead! (blocked entirely)

**Comment by Aka-but-nice** [comment]
This method is old, try #347 or #348, and ik I sound like an advertisement saying to view their product

**Reply by rwg100** [reply]
still works for me tho and im on v108
also you supposed to add your school acc back not a home acc

**Reply by Aka-but-nice** [reply]
Ik it still works

**Comment by Aka-but-nice** [comment]
also what json

**Comment by 3kh0** [comment]
Use 3kh0.github.io for games 😎

**Reply by Aka-but-nice** [reply]
@3kh0 bro I don't think not one time when I went on your site, I clicked on games

**Comment by Aka-but-nice** [comment]
@shrimpledimple what json?
